src/app/services/auth/auth.service.ts

```diff
@@ -30,27 +30,29 @@ export class AuthService {
     private router: Router,
     location: Location,
   ) {
-    this.oauthService.configure(environment.authConfig);
-    this.oauthService.setupAutomaticSilentRefresh();
-    // If the user should not be forcefully logged in (e.g. if you have pages, which can be
-    // accessed anonymously), change loadDiscoveryDocumentAndLogin to
-    // loadDiscoveryDocumentAndTryLogin and have a login functionality in the
-    // template of the component injecting the AuthService which calls the login() method.
-    this.initialized = this.oauthService
-      .loadDiscoveryDocumentAndLogin({state: location.path()})
-      // If the user is not logged in, he will be forwarded to the identity provider
-      // and this promise will not resolve. After being redirected from the identity
-      // provider, the login promise will return true.
-      .then((v) => (v ? true : new Promise(() => {})));
-    // Redirect the user to the url configured with state above or in a separate login call.
-    this.oauthService.events
-      .pipe(first((e) => e.type === "token_received"))
-      .subscribe(() => {
-        const state = decodeURIComponent(this.oauthService.state || "");
-        if (state && state !== "/") {
-          this.router.navigate([state]);
-        }
-      });
+    if (!environment.disableBackend) {
```

Comment:
This could be simplified with an early return.

Reply:
agreed and fixed